lib/utils/camel-case.ts

```diff
@@ -0,0 +1,19 @@
+export function camelCase(s: string) {
+  if (!s) return '';
+  // split camel case, then match ascii words
+  const words = asciiWords(s.replace(/([a-z0-9])([A-Z])/g, '$1 $2'));
+  return words.length > 1
```

Comment:
I think you should have access to camelCase already through the yargs-parser dependency:

https://github.com/yargs/yargs-parser/blob/main/lib/string-utils.ts#L7

I believe I already use one or two methods off of it elsewhere.